scenarios/aws/fakeintake/fakeintake.go

```diff
@@ -65,7 +65,7 @@ func NewECSFargateInstance(e aws.Environment, name string, option ...Option) (*f
 		if err != nil {
 			return err
 		}
-
+		e.Ctx.Log.Info(fmt.Sprintf("Fakeintake dashboard available at: https://dddev.datadoghq.com/dashboard/xzy-ybs-wz4/e2e-tests--fake-intake?fromUser=true&tpl_var_fake_intake_task_family[0]=%s-fakeintake-ecs", e.Ctx.Stack()), nil)
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**